src/shipit_static_analysis/shipit_static_analysis/revisions.py

```diff
@@ -3,13 +3,67 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import re
+from shipit_static_analysis.config import settings
+from parsepatch.patch import Patch
 from cli_common import log
+import hglib
+import os
+import re
 
 logger = log.get_logger(__name__)
 
 
-class PhabricatorRevision(object):
+class Revision(object):
+    '''
+    A common mercurial revision
+    '''
+    mercurial = None
+    files = []
+    lines = {}
+
+    def analyze_files(self, repo):
```

Comment:
I assume nothing has changed here and the code was just moved

Reply:
Exactly, i did that to get access to `has_clang _files` which gives us a more readable code.